package.json

```diff
@@ -68,6 +68,7 @@
   },
   "scripts": {
     "start": "node server.js",
+    "postinstall": "mkdir -p ./bench/data && curl -s https://d2ad6b4ur7yvpq.cloudfront.net/naturalearth-3.3.0/ne_50m_land.geojson > ./bench/data/naturalearth-land.json",
```

Comment:
We should not download megabytes of fixtures for every `mapbox-gl` install. This data is only needed for developers.